include/st_api.h

```diff
@@ -206,6 +206,15 @@ static inline bool st_is_frame_complete(enum st_frame_status status) {
  */
 double st_frame_rate(enum st_fps fps);
 
+/**
+ * Helper function returning enum st_fps from frame rate
+ * @param framerate
+ *   frame rate number
+ * @return
+ *   enum st_fps fps.
+ */
+enum st_fps st_frame_rate_to_st_fps(int framerate);
```

Comment:
not sure how a int value can represent fps? why not use a more readable float/double type?

Reply:
the original implementation is unsigned int fps = framerate.num * 100 / framerate.den; which results in an int value

Reply:
Can we change it? Hard to read what is the mean of `unsigned int fps = framerate.num * 100 / framerate.den`, use `double fps = framerate.num / framerate.den` is more readable and even we can print the fps directly.